src/mame/juku/juku.cpp

```diff
@@ -132,17 +164,20 @@ void juku_state::bank_map(address_map &map)
 
 void juku_state::io_map(address_map &map)
 {
-	map(0x00, 0x01).rw(m_pic, FUNC(pic8259_device::read), FUNC(pic8259_device::write));
+	map(0x00, 0x03).rw(m_pic, FUNC(pic8259_device::read), FUNC(pic8259_device::write));
```

Comment:
The 8259 only has one address line (A0), did you mean to mirror the accesses? In that case use `mirror(0x02)`.

Reply:
This was a mistake. Reverted.